gk/main.c

```diff
@@ -634,6 +711,34 @@ gk_proc(void *arg)
 			process_gk_cmd(gk_cmds[i], instance);
 			mb_free_entry(&instance->mb, gk_cmds[i]);
         	}
+
+		if (ipv6_if_configured(&gk_conf->net->front) && acl.num > 0) {
+			ret = rte_acl_classify(
```

Comment:
There's `rte_acl_init()` in `dependencies/dpdk/lib/librte_acl/rte_acl.c` that detects the best algorithm available, but for some reason `rte_acl_init()` is `static` and never called. Isn't asking in the mailing list the why worth?